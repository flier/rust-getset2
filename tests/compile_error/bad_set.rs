use getset2::Setter;

#[derive(Setter)]
#[set(foobar)]
struct Foobar {}

fn main() {}
