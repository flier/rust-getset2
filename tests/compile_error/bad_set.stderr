error: invalid #[set(..)] attribute, cannot find parameter `foobar` in this scope
 --> tests/compile_error/bad_set.rs:4:1
  |
4 | #[set(foobar)]
  | ^
